site/docs/develop/java.md

```diff
@@ -131,8 +131,8 @@ config.getVersion();
 Creates a new commit by adding metadata for an `IcebergTable` under the specified `ContentKey` instance represented by `key` and deletes content represented by `key2`
 
 ```java
-ContentKey key = ContentKey.of("table.name.space", "name");
-ContentKey key2 = ContentKey.of("other.name.space", "name2");
+ContentKey key = ContentKey.of("<table.namespace>", "<table.name>");
+ContentKey key2 = ContentKey.of("<other.namespace>", "<other.name>");
```

Comment:
Why would you like to add `<` `>` brackets to keys?

Reply:
Just to let users know that these are placeholders.

Reply:
Hmmm... not sure I follow. This example is compilable and runnable code "as is", without replacing any placeholders.

Reply:
I agree the code is runnable, but in order to get any proper output from this piece of code, namespace and table name needs to be replaced with actual names. That is exactly what I am trying to highlight here. :) 

But still if you feel otherwise, I will revert this change. Please let me know.

Reply:
Maybe use something like `ContentKey.of("your-namespace", "your-table-name")`

Reply:
Yes, special chars are best avoided in "Getting Started" examples (`.`, `<`, `>` in this case).

Reply:
Changed.